src/jit/morph.cpp

```diff
@@ -12791,6 +12791,12 @@ GenTree* Compiler::fgMorphSmpOp(GenTree* tree, MorphAddrContext* mac)
 
         case GT_NOT:
         case GT_NEG:
+            // Remove double negation
+            if (oper == GT_NEG && op1->OperIs(GT_NEG))
```

Comment:
Please change 
`if (oper == GT_NEG && op1->OperIs(GT_NEG))`
to
`if ((oper == GT_NEG) && op1->OperIs(GT_NEG))`
per our convention.

Reply:
Fixed!